requirements/common.txt

```diff
@@ -18,6 +18,8 @@ plotly==5.17.0
 pprintpp==0.4.0
 psutil>=5.7.0
 pymongo>=3.12
+pydantic==2.6.4
```

Comment:
Stuard had removed `pydantic` in the past. Is bringing back here intentional/needed?